tests/copy.c

```diff
@@ -134,10 +134,6 @@ static struct xnvme_cli_sub g_subs[] = {
 			{XNVME_CLI_OPT_POSA_TITLE, XNVME_CLI_SKIP},
 			{XNVME_CLI_OPT_URI, XNVME_CLI_POSA},
 
-			{XNVME_CLI_OPT_NON_POSA_TITLE, XNVME_CLI_SKIP},
-			{XNVME_CLI_OPT_SLBA, XNVME_CLI_LOPT},
-			{XNVME_CLI_OPT_ELBA, XNVME_CLI_LOPT},
-
```

Comment:
Looks ok, but I am thinking does it makes sense to fix the `test_copy` function so that user can pass slba, sdlba and nlb. Maybe we can have some sort of check like this so that slba and sldba don't overlap
```
if (slba < sdlba)
    assert (slba + nlb <= sldba);
else
    assert (sldba + nlb <= slba);
```

Reply:
@ankit-sam I have tried to address this now